docs/hardware_setup.md

```diff
@@ -49,9 +49,9 @@ It is important to remember that when launching the VIO, the camera should be st
 
 3. Calibrate camera intrinsics and extrinsics using [Kalibr](https://github.com/ethz-asl/kalibr), recommended model is: ```pinhole-equi``` [(see OpenCV documentation)](https://docs.opencv.org/3.3.1/db/d58/group__calib3d__fisheye.html)
 
-4. Create configuration files for KimeraVIO ROS wrapper using [Kalibr2KimeraVIO-pinhole-equi](https://github.mit.edu/SPARK/VIO/blob/feature/parallelization/jpl/kalibr/kalibr2sparkvio_stereo_pinhole-equi.py) or [[Kalibr2KimeraVIO-pinhole-radtan](https://github.mit.edu/SPARK/VIO/blob/feature/parallelization/jpl/kalibr/kalibr2sparkvio_stereo_pinhole-equi.py)]
+4. Create configuration files for KimeraVIO ROS wrapper using [Kalibr2KimeraVIO](https://github.com/MIT-SPARK/Kimera-VIO/kalibr2kimeravio.py)
 
-5. Create/adapt your own specific launch file, similar to [example MyntEye S](https://github.mit.edu/SPARK/spark_vio_ros/blob/jpl/launch/spark_vio_ros_mynteye.launch)
+5. Create/adapt your own specific launch file, similar to [example MyntEye S](https://github.com/MIT-SPARK/Kimera-VIO-ROS/kimera_ros_mynteye.launch)
```

Comment:
This address returns 404, Should probably point to: https://github.com/MIT-SPARK/Kimera-VIO-ROS/blob/master/launch/kimera_ros_mynteye.launch ?

Reply:
Yes thanks! Updating now.